da/executor/executor.py

```diff
@@ -0,0 +1,49 @@
+import asyncio
+import proto
+from transport import Transport
+
+class Executor:
+    def __init__(self, addr, port, col_num):
+        self.addr = addr
+        self.port = port
+        self.col_num = col_num
+        self.connections = []
+        self.interval = 10
+
+    async def execute(self):
+        blob_id = b"test"
+        data = b"TEST DATA"
+        while True:
+            try:
+                for transport in self.connections:
+                    message = proto.new_dispersal_req_msg(blob_id, data)
+                    await transport.write(message)
```

Comment:
Nitpick: Did you try to write it councurrently?

Reply:
You mean something like that?:
```python
    async def execute(self):
        blob_id = b"test"
        data = b"TEST DATA"
        while True:
            try:
                tasks = []
                for transport in self.connections:
                    message = proto.new_dispersal_req_msg(blob_id, data)
                    task = asyncio.create_task(transport.write(message))
                    tasks.append(task)
                await asyncio.gather(*tasks)
                await asyncio.sleep(self.interval)
            except asyncio.CancelledError:
                break
            except Exception as e:
                print(f"Executor: Error during message sending: {e}")
```

Reply:
Yeah, something like that would do. Or in an `async for` using `asyncio.as_completed` if we need the result.